api/ruvnl-morning.ts

```diff
@@ -13,36 +13,38 @@ const getOcfDAForecastCsv: (source: "wind" | "solar", token: string) => Promise<
   });
 }
 
-const sendQuartzEmail: (resend: Resend, recipients: string[], subject: string, filename: string, content: Buffer) => Promise<CreateEmailResponse> = async (resend, recipients, subject, filename, content) => {
+const sendQuartzEmails: (resend: Resend, recipients: string[], subject: string, filename: string, content: Buffer) => Promise<CreateBatchResponse> = async (resend, recipients, subject, filename, content) => {
   const html = "<span>Good morning,<br/><br/>" +
     "Find attached the OCF Day Ahead forecast for tomorrow.<br/><br/>" +
     "Kind regards,<br/>" +
     "The Open Climate Fix Team" +
     "<br/><br/><br/></span>";
 
-  return await resend.emails.send({
-    from: 'Quartz Energy <notifications@mail.quartz.energy>',
-    reply_to: "quartz.support@openclimatefix.org",
-    to: recipients,
-    subject,
-    html,
-    attachments: [
-      {
-        filename,
-        content,
-        content_type: 'text/csv; charset="UTF-8"',
-      },
-    ],
-    tags: [
-      {
-        name: 'category',
-        value: 'ruvnl_email',
-      },
-    ],
-  });
+  return await resend.batch.send(recipients.map(recipient => {
```

Comment:
could you add a comment why you are using batch, like 
`We want to send a separate email to each person, rather than one email to everyone. `